torch/jit/__init__.py

```diff
@@ -354,22 +356,38 @@ def __getattr__(self, attr):
         return self.module._get_method(attr)
 
 
-def _script_graph(fn, _frames_up=0):
+def _compile_fn(fn, _frames_up=0):
+    mod = ScriptModule()
+    rcb = createResolutionCallback(_frames_up + 1)
+    ast = get_jit_ast(fn)
+    arg_types, ret_type = annotations.get_signature(fn, ast.num_params(), 0)
+    _jit_script_compile(mod, torch._C.DefAndTypes(ast, arg_types, ret_type), rcb)
+    return mod
+
+
+def _script_pure_function(mod, fn, _frames_up=0):
     rcb = createResolutionCallback(_frames_up + 1)
     ast = get_jit_ast(fn)
-    return _jit_script_compile(ast, rcb)
+    arg_types, ret_type = annotations.get_signature(fn, ast.num_params(), 0)
+    _jit_script_compile_pure_fn(mod, torch._C.DefAndTypes(ast, arg_types, ret_type), rcb)
+
+
+def _script_graph(fn, _frames_up=0):
```

Comment:
Is this still used somewhere? It's not much different from `script(fn).graph`, is it? I'd just remove it